lints/lint_name_constraint_on_edi_party_name.go

```diff
@@ -19,19 +19,19 @@ import (
 	"github.com/zmap/zlint/util"
 )
 
-type ncOnEdi struct {
+type ncOnEDI struct {
```

Comment:
you might spell out nc as name constraint. That's not an acronym that's generally used in this space.